modules/cleanup/configure.sh

```diff
@@ -2,4 +2,4 @@
 # Configure module
 set -e
 
-rm -rf "$JBOSS_HOME"
```

Comment:
shouldn't you stil remove the `$JBOSS_HOME` dir before creating the symlink?

Aiui, there must still be nothing at `$JBOSS_HOME` in the runtime image so that the server can be copied at this location when the application image is built

Reply:
The removal was done to workaround an issue during image build. We are now defining JBOSS_HOME in this module, so JBOSS_HOME is undefined prior to reach this point, so no JBOSS_HOME exists.